### PR TITLE
code of conduct good

### PR DESCRIPTION
**Pull request by tesinormed**
@Team-MythosCraft/mythoscraft-fabric-devs please review

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at unknownperson102424@pm.me. All
```
Comment (TheOnlyTails) [inline CODE_OF_CONDUCT.md]: @DenDragon23 , What email do you want here?

Comment (tesinormed) [reply CODE_OF_CONDUCT.md]: yes that is my email

**Comment by tesinormed** [comment]
code review suggestions have been resolved, please re-review

**Review by TheOnlyTails — APPROVED** [review]
Looks good to me! Just don't forget to add this to the forge repo too.

**Pull request merged**